modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1911,6 +1911,11 @@ return view.extend({
 				]).then(this.renderContents.bind(this));
 			};
 
+			if (bss.ssid == null) {
+				name = s2.option(form.Value, 'ssid', _('Network SSID'), _('The correct SSID must be manually specified when joining a hidden wireless network'));
+				name.optional = false
```

Comment:
Can you add a semicolon here (sorry to be pedandic about that :) )

Reply:
np i'm stupid that i miss think kind of things...